lib/bundles/inspec-habitat/profile.rb

```diff
@@ -336,23 +332,21 @@ def run_hook_contents
       <<-EOL
 #!/bin/sh
 
-export PATH=${PATH}:$(hab pkg path core/ruby)/bin
-
 # InSpec will try to create a .cache directory in the user's home directory
 # so this needs to be someplace writeable by the hab user
 export HOME={{pkg.svc_var_path}}
 
-PROFILE_IDENT="#{habitat_origin}/#{package_name}"
+PROFILE_IDENT="{{pkg.origin}/{{pkg.name}}"
```

Comment:
Missing a `}` for `{{pkg.origin}}`... without it, the sup fails with:

```
Failed to load hook: hab-sup(ER)[src/error.rs:355:8]: TemplateError(TemplateError { reason: InvalidSyntax, template_name: Some("hook"), line_no: Some(7), column_no: Some(28) })
```

... and then the sup fails because the run hook never got copied to `/hab/svc`

```
Service start failed: hab-sup(ER)[src/error.rs:379:8]: No such file or directory (os error 2)
```